src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotChangeEventSource.java

```diff
@@ -366,8 +370,11 @@ protected SnapshotResult<YugabyteDBOffsetContext> doExecute(ChangeEventSourceCon
 
                 OpId cp = previousOffset.snapshotLSN(tabletId);
 
-                LOGGER.debug("Going to fetch from checkpoint {} for tablet {} for table {}", 
-                            cp, tabletId, table.getName());
+                if (LOGGER.isDebugEnabled() || System.currentTimeMillis() >= (lastLoggedTimeForGetChanges + 300_000)) {
```

Comment:
Always use a config for these so that we can enable/disable. Please add a config.